incubator/nodejs-express/image/project/package.json

```diff
@@ -15,13 +15,13 @@
   },
   "dependencies": {
     "@cloudnative/health-connect": "^2.0.0",
-    "appmetrics-prometheus": "^3.0.0",
+    "appmetrics-prometheus": "github:sam-github/appmetrics-prometheus#optional-patching",
```

Comment:
Hey @sam-github, spoke with @andrewhughes101 about this PR. Let's wait until https://github.com/CloudNativeJS/appmetrics-prometheus/pull/35 is in from the appmetrics prometheus side so we can use an officially released version here.